CMakeLists.txt

```diff
@@ -4,29 +4,29 @@ cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 # The cmake_minimum_required(VERSION) command implicitly invokes the cmake_policy(VERSION)
 # command to specify that the current project code is written for the given range of CMake
 # versions.
-project(lxqt-globalkeys)
+project(lxqt2-globalkeys)
```

Comment:
Why changing `lxqt` to `lxqt2` here? `lxqt-globalkeys` is a component that should be replaced, not installed in parallel to its Qt5 version.

Reply:
That's a leftover from the beginning, atm panel looks for this.

Reply:
> atm panel looks for this.

Yes, I noticed it in the panel port first. But future reviews will be done from top to bottom.

Reply:
I've removed every "2" suffix, including renamed translation files. This is now "2" free version.